package.json

```diff
@@ -123,39 +123,39 @@
     "@types/express-session": "^1.17.4",
     "@types/http-errors": "^1.8.1",
     "@types/jest": "^27.0.1",
-    "@types/jsonwebtoken": "^8.5.4",
-    "@types/node": "^14.17.3",
+    "@types/jsonwebtoken": "^8.5.5",
+    "@types/node": "^14.17.14",
     "@types/nunjucks": "^3.1.5",
     "@types/passport": "^1.0.7",
     "@types/passport-oauth2": "^1.4.11",
     "@types/redis": "^2.8.31",
     "@types/superagent": "^4.1.12",
     "@types/supertest": "^2.0.11",
-    "@typescript-eslint/eslint-plugin": "^4.29.1",
-    "@typescript-eslint/parser": "^4.29.1",
+    "@typescript-eslint/eslint-plugin": "^4.30.0",
+    "@typescript-eslint/parser": "^4.30.0",
     "concurrently": "^6.2.1",
-    "cypress": "^8.2.0",
+    "cypress": "^8.3.1",
     "cypress-multi-reporters": "^1.5.0",
     "eslint": "^7.32.0",
     "eslint-config-airbnb-base": "^14.2.1",
     "eslint-config-prettier": "^8.3.0",
     "eslint-import-resolver-typescript": "^2.4.0",
     "eslint-plugin-cypress": "^2.11.3",
-    "eslint-plugin-import": "2.24.0",
-    "eslint-plugin-prettier": "^3.4.0",
-    "husky": "^7.0.1",
-    "jest": "^27.0.6",
+    "eslint-plugin-import": "2.24.2",
+    "eslint-plugin-prettier": "^4.0.0",
```

Comment:
Take it there's no breaking changes here?

Reply:
looking at https://github.com/prettier/eslint-plugin-prettier/compare/v3.3.1...v4.0.0 - only thing I could see was dropping support for older versions of Node - runs fine locally 😄